bin/lib/zopen-download

```diff
@@ -1,5 +1,5 @@
 #!/bin/sh
-# Downloads and extracts the latest ZOSOpenTools releases from GitHub
+# Package Manager for z/OS Open Tools - https://github.com/ZOSOpenTools
```

Comment:
not sure we should call this a 'package manager'. How about something in-between 'downloads and extracts' and 'package manager' :)

Reply:
Changed it to download utility